xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -304,7 +304,7 @@ void CRenderManager::FrameMove()
       lock.unlock();
       if (!Configure())
         return;
-
+      UpdateLatencyTweak();
```

Comment:
in which scenario does this help?

Reply:
I found HDR mode is switched on only after renderer is configured (line 305) so it's always necessary to add HDR latency.

I don't know if we can get rid of another call to UpdateLatencyTweak() without side-effect. I can test it to the extent of my ability to understand this part of the code, I didn't feel confident to remove it.

Reply:
OK. That fits how Windows/Xbox work but I don't know for the other platforms.

Reply:
HDR should be set at this stage. At least it does for Linux GLES renderer aswell.

That doesn't matter anyway since "GetOSHDRStatus()" seems only implemented for Windows. That's odd I didn't expect that.